GREEDY/Controllers/OCRController.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using GREEDY.Models;
+using System.Drawing;
+using Tesseract;
+
+namespace GREEDY.Controllers
+{
+    public class OCRController
+    {
+        private Receipt receipt;
```

Comment:
Could be a bit confused between Receipt and receipt

Reply:
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members
The coding standard says otherwise (CONSIDER giving a property the same name as its type.)